catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/QueryImpl.java

```diff
@@ -163,6 +162,21 @@ public void setRequestsTotalResultsCount(boolean requestsTotalResultsCount) {
 
   @Override
   public String toString() {
-    return ToStringBuilder.reflectionToString(this);
+    return "QueryImpl{"
```

Comment:
❓ Was this changed from StringBuilder because the compiler will implicitly use StringBuilder anyways?

Reply:
Yes. I just took the IDEA generated toString as that is what we use other places and since StringBuilder is used under the covers, I figured it was best to use that.